pkg/actions/project.go

```diff
@@ -291,9 +291,9 @@ func ProjectLinkList(c *cli.Context) {
 		} else {
 			w := new(tabwriter.Writer)
 			w.Init(os.Stdout, 0, 8, 2, '\t', 0)
-			fmt.Fprintln(w, "ENVIRONMENT VARIABLE \tPROJECT ID")
+			fmt.Fprintln(w, "PROJECT NAME \tENVIRONMENT VARIABLE")
```

Comment:
Could we add the url here? Maybe as `Target URL` otherwise the only way to get that for debugging is to use the JSON mode.
(Also Target Project might be a better name for the first column to make it clear it's not the current project.)